src/syft/core/common/message.py

```diff
@@ -46,15 +55,14 @@ class SignedMessage(SyftMessage):
 
     obj_type: str
     signature: bytes
-    verify_key: bytes
-    message: bytes
+    verify_key: VerifyKey
 
     def __init__(
         self,
         address: Address,
         obj_type: str,
         signature: bytes,
-        verify_key: bytes,
+        verify_key: VerifyKey,
```

Comment:
Very good - glad you stuck with this. When we were passing this as bytes before we were conflating application logic and serialiation a bit too much. This looks good.

Reply:
I actually found this writing a test since they weren’t equal after deserialising so I think it wasn’t balanced before.